README.md

```diff
@@ -26,6 +26,72 @@ Run prettier to lint and write - this is also done automatically in a pre-commit
 yarn lint
 ```
 
+## Deploying to remote networks
+Requirement: `gcloud` must be set up and user account must have access staked-celo-alfajores and staked-celo-staging on GCP. 
+Next, ensure the environment variables have been decrypted, using `yarn keys:decrypt`.
+Then use the following commands to deploy depending on the desired target environment.
+For example, the below command will deploy to the Alfajores network, using the decrypted private key, the default Alfajores rpc url and the other variables in `.env.alfajores`. 
+
+Alfajores : 
+```
+yarn deploy --network alfajores
+```
+
+You may immediately verify the deployment, with the following commands.
+
+Alfajores : 
+```
+yarn verify --network alfajores
+```
+
+You may desire to deploy using an unlocked account in a private node. In that case, you can use the following commands :
+
+```
+yarn hardhat [GLOBAL OPTIONS] stakedCelo:deploy --from <STRING> --tags <STRING> --url <STRING> [--use-private-key]
+```
+
+example
+```
+yarn hardhat stakedCelo:deploy  --network alfajores --show-stack-traces --tags core  --url "http://localhost:8545" --from "0xff2694d968246F27093D095D8160E005D5f31e5f" --use-private-key
+```
+ 
+Run `yarn hardhat help stakedCelo:deploy` for more information.
+
+You may want to run your deployment via a light node using an unlocked account, in that case follow the steps below
+### Steps to Run a light node:
+
+Step 1: Create and fund an account.
+In your terminal, run the below command
+
+```
+export $ALFAJORES_CELO_IMAGE=us.gcr.io/celo-org/geth:alfajores
+```
+
+Then create a directory called `celo-tools` , cd into it and run the below command:
```

Comment:
nit: `celo-data-dir` instead of `celo-tools`?

I got confused for a second thinking we were using `celotools` from monorepo